.github/workflows/docker-build-test.yaml

```diff
@@ -25,6 +25,16 @@ on: # build on main branch OR when a PR is labeled with `CICD:build-images`
   workflow_dispatch:
   pull_request_target:
     types: [labeled, opened, synchronize, reopened, auto_merge_enabled]
+    # For most PRs run workflows from main, not the PR commit for security
+    paths-ignore:
+      - 'documentation/**'
+      - '.github/**'
+  # For PR that modify .github, run from that PR
+  # This will fail to get secrets if you are not from aptos
+  pull_request:
```

Comment:
Do you know how this will affect the branch protection rules which wait for success based on job name?

<img width="840" alt="image" src="https://user-images.githubusercontent.com/12578616/192391931-ccf951ce-e36d-4832-9f97-10013b94b8f6.png">

Reply:
I have no idea how this will work, however this should only be triggered when people are making workflow changes which should not be so frequent outside of forge development